examples/specs/scatter_opacity_yfield.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Duplicated examples are deleted.
I checked every existed examples with `config.mark.opacity` and changed them to `encoding.opacity.value` and they all work well.